cob_moveit_bringup/bin/add_robot.sh

```diff
@@ -2,10 +2,10 @@
 
```

Comment:
They got lost in https://github.com/ipa320/cob_manipulation/commit/763ed4e1dc0b0987db5a8a70262836e49baa01a8